notifications_utils/clients/redis/bounce_rate.py

```diff
@@ -30,6 +31,10 @@ class RedisBounceRate:
     def __init__(self, redis: RedisClient):
         self._redis_client = redis
 
+    def init_app(self, app, *args, **kwargs):
+        self._critical_threshold = app.config.get("BR_CRITICAL_PERCENTAGE")
+        self._warning_threshold = app.config.get("BR_WARNING_PERCENTAGE")
```

Comment:
```suggestion
        self._critical_threshold = app.config.get("BR_CRITICAL_PERCENTAGE") if app.config.get("BR_CRITICAL_PERCENTAGE") else BR_CRITICAL_PERCENTAGE_DEFAULT
        self._warning_threshold = app.config.get("BR_WARNING_PERCENTAGE") if app.config.get("BR_WARNING_PERCENTAGE") else BR_WARNING_PERCENTAGE_DEFAULT
```

Could we do something like this to set defaults in this file?

Reply:
Added!